substrate-node/pallets/pallet-validator/src/mock.rs

```diff
@@ -1,7 +1,6 @@
-use super::*;
 use crate as pallet_validator;
 use core::cell::RefCell;
-use frame_support::{bounded_vec, construct_runtime, parameter_types, traits::ConstU32};
+use frame_support::{bounded_vec, construct_runtime, parameter_types, traits::{ConstU32, GenesisBuild}};
```

Comment:
this line should give this
~~~rust
use frame_support::{
    bounded_vec, construct_runtime, parameter_types,
    traits::{ConstU32, GenesisBuild},
};
~~~
with automatic formatting

Reply:
done